x-pack/test/api_integration/apis/security/api_keys.ts

```diff
@@ -65,6 +67,33 @@ export default function ({ getService }: FtrProviderContext) {
             expect(name).to.eql('test_api_key_with_metadata');
           });
       });
+
+      it(`${basic ? 'basic' : 'trial'} license should ${
+        basic ? 'not allow' : 'allow'
+      } a cross cluster API Key to be created`, async () => {
+        const result = await supertest
+          .post('/internal/security/api_key')
+          .set('kbn-xsrf', 'xxx')
+          .send({
+            type: 'cross_cluster',
+            name: 'test_cc_api_key',
+            metadata: {},
+            access: {
+              search: [
+                {
+                  names: ['logs*'],
+                  query: { bool: { must_not: { term: { field2: 'value2' } } } },
+                  field_security: { grant: ['field2'] },
+                  allow_restricted_indices: true,
+                },
+              ],
+            },
+          });
+        expect(result.status).to.be(basic ? 403 : 200);
+        if (!basic) {
+          expect(result.body.name).to.be('test_cc_api_key');
+        }
+      });
```

Comment:
there is error case here too. however, not sure if you need to test it or not since ES owns the error logic.

if you define (`access.search` and `access.replication`) where `access.search` has either (`access.search.query` or `access.search.field_security`) the request will be rejected with an error.

Reply:
We typically let any ES-specific error logic fall through, but I could add a check expecting ES to reject under these conditions if we think it adds value to have some redundancy in our testing.

Reply:
Added tests in 39098db30370ab20f5aa6bbe842dd6e928308049